components/article-list-item.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Link from 'next/link';
+import { string } from 'prop-types';
+
+const Item = (props) =>{
+  return (
+    <div className="card">
```

Comment:
Maybe use HTML's <article> tag for that :)

Reply:
For me, it's only a list item, a link to the article, not an article itself. Is it ok to use Article tag in such contexts?